lib/ui/pages/note_camera/note_camera.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:camera/camera.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:get/get.dart';
+import 'package:flutter_app/src/store/picture_storage.dart';
+
+class NoteCamera extends StatefulWidget {
+  final String courseId;
+
+  const NoteCamera({Key? key, required this.courseId}) : super(key: key);
+
+  @override
+  State<NoteCamera> createState() => _NoteCameraState();
+}
+
+class _NoteCameraState extends State<NoteCamera> with WidgetsBindingObserver {
+  CameraController? controller;
+  bool _isCameraInitialized = false;
+  late List<CameraDescription> cameras;
```

Comment:
Suggest assigning an empty list instead of relying on `late`.

```suggestion
  List<CameraDescription> cameras = const [];
```